frontend/src/components/CookiesContainer.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { useCallback, useState } from 'react';
+import { Notification } from 'wix-style-react';
+import cookies from '../utils/cookies';
+
+const CookiesContainer = () => {
+  const isAccepted = cookies.get('acceptCookiesPolicy');
+
+  const [, updateState] = useState();
+  const forceUpdate = useCallback(() => updateState({}), []);
+
+  const onAccept = () => {
+    cookies.set('acceptCookiesPolicy', 'accepted', { path: '/', sameSite: true });
+    forceUpdate();
```

Comment:
why `forceUpdate` instead of having:
`const [isAccepted, setIsAccepted] = useState(cookies.get(..)); ?